Scripts/RemediationScripts/Readme.md

```diff
@@ -1364,6 +1364,25 @@ Yes
 
 ___
 
+## 66. Azure_BackupVault_DP_Enable_Soft_Delete_Trial
+
+### Display Name
+Always-on soft delete must be enabled on Backup Vault
+
+### Link to Bulk Remediation Script (BRS)
+[Remediate-BackupVaultSoftdelete](Remediate-BackupVaultSoftdelete.ps1)
+
+### Minimum permissions required to run the script
+Owner role at subscription level
```

Comment:
Do we Owner role at subscription scope?

Reply:
Updated to Contributor role on resource level.
Note: On UI we can access those properties(soft delete & immutability) only if we have owner access on subscription level. But through powershell command contributor or higher access at resource level is sufficient to update properties.